README.md

```diff
@@ -1,10 +1,8 @@
-Clash - A functional hardware description language
-==================================================
+# Clash - A functional hardware description language
 
-[![Hackage Release Version](https://img.shields.io/hackage/v/clash-ghc.svg)](https://hackage.haskell.org/package/clash-ghc)
-[![Hackage Dependency Freshness](https://img.shields.io/hackage-deps/v/clash-ghc.svg?style=flat)](https://packdeps.haskellers.com/feed?needle=exact%3Aclash-ghc)
-[![Documentation Status](https://readthedocs.org/projects/clash-lang/badge/?version=latest)](https://clash-lang.readthedocs.io/en/latest/?badge=latest)
```

Comment:
Do we not want this badge?

Reply:
Super useful highlighting from GitHub. I mean the readthedocs badge

Reply:
We do! Sorry

Reply:
Okay, if you add that back I'll approve it

Reply:
Done.